user_manual/groupware/calendar.rst

```diff
@@ -2,28 +2,47 @@
 Using the Calendar app
 ======================
 
-.. note:: The calendar app is not enabled by default and needs to be installed
-          separately from our App Store. Please ask your Administrator for it.
+.. note:: The Calendar app comes installed with Nextcloud Hub by default, but can be disabled.
```

Comment:
```suggestion
.. note:: The Calendar app comes installed with Nextcloud Hub by default, which can be disabled.
```

Reply:
this doesn't make sense IMO. then it sounds like you can disable that the app comes by default